pandas/tseries/resample.py

```diff
@@ -324,6 +326,11 @@ def _resample_timestamps(self, kind=None):
 
                 result.index = result.index + loffset
 
+        if kind == "timedelta" and len(result.index) > 0:
```

Comment:
I *think* you can just do the calculation in ``_get_time_delta_bins``. ``base`` is just added to the labels (but not the binner).